tsconfig.json

```diff
@@ -19,6 +19,7 @@
     "resolveJsonModule": true,
     "noUnusedParameters": true,
     "downlevelIteration": true,
+    "useUnknownInCatchVariables": false,
```

Comment:
This is indeed a great TS feature, but it will require reviewing every exception and typing them correctly. This is out of the scope of this change.